src/test/java/com/egov/business-services/tests/egfMasterChartOfAccount.feature

```diff
@@ -0,0 +1,108 @@
+Feature: Chart of account
+Background:
+  * def jsUtils = read('classpath:jsUtils.js')
+  * def glcode = ranInteger(6)
+  * def name = ranString(5) + "Bank"
+  * def egfMasterChartOfAccountConstant = read('../../business-services/constants/egfMasterChartOfAccount.yaml')
+  # Required parameters to create acccount
+  * def isActiveForPosting = 'false'
+  * def functionRequired = 'false'
+  * def budgetCheckRequired = 'false'
+  * def classification = '1'
+  * def type = 'B'
+  * def invalidTenantId = ranString(3)
+
+@ChartOfAccountCreate_01 @chartOfAccountCreate  @chartOfAccount
+Scenario: Verify creating chart of accounts through API call
+* call read('../../business-services/preTests/egfMasterChartOfAccountCreate.feature@createAccountSuccessfully')
+* match chartOfAccountCreateResponseBody.chartOfAccounts.length == '##[_ > 0]'
+
+@ChartOfAccountCreate_UniqueBank_02  @chartOfAccountCreate  @chartOfAccount
+Scenario: Verify creating chart of accounts by passing a bank name that already exists and check for errors
+* call read('../../business-services/preTests/egfMasterChartOfAccountCreate.feature@createAccountSuccessfully')
+* def name = chartOfAccountCreateResponseBody.chartOfAccounts[0].name
```

Comment:
Please provide inline comments. What exactly you are trying to fetch and save on `name`